### PR TITLE
[lldb] Skip validation of CGPoint in TypeSystemSwiftTypeRef

### PR DESCRIPTION
**Pull request by augusto2112**

### CODE REVIEW

**Comment by augusto2112** [comment]
@swift-ci test

**Review by kastiglione — APPROVED** [review]

File: lldb/source/Plugins/TypeSystem/Swift/TypeSystemSwiftTypeRef.cpp [hunk 1]
```diff
@@ -3857,6 +3857,9 @@ bool TypeSystemSwiftTypeRef::ShouldSkipValidation(opaque_compiler_type_t type) {
   if (mangled_name == "$sSo11CFStringRefaD")
     return true;
 
+  if (mangled_name == "$sSo7CGPointVD")
+    return true;
+
```
Comment (clackary) [inline lldb/source/Plugins/TypeSystem/Swift/TypeSystemSwiftTypeRef.cpp:3859]: Looks like CGSize also trips the same assertion, and I'm guessing there could be more.

```suggestion
  if (mangled_name == "$sSo7CGPointVD")
    return true;

  if (mangled_name == "$sSo6CGSizeVD")
    return true;
```

Comment (clackary) [reply lldb/source/Plugins/TypeSystem/Swift/TypeSystemSwiftTypeRef.cpp:3859]: Actually, the test succeeded with just those two.

**Comment by augusto2112** [comment]
@swift-ci test

**Comment by JDevlieghere** [comment]
@swift-ci test macos

**Comment by augusto2112** [comment]
Error seems to be unrelated:

```
clang-13: error: no such file or directory: 'stdlib/public/Cxx/std/OSX/arm64/std.o'
```

**Comment by augusto2112** [comment]
@swift-ci test macOS

**Comment by augusto2112** [comment]
@swift-ci test macOS

**Comment by JDevlieghere** [comment]
@swift-ci test macOS

**Comment by augusto2112** [comment]
https://github.com/apple/swift/pull/59955

@swift-ci please test macOS

**Pull request merged**